src/main/clojure/clara/rules/accumulators.cljc

```diff
@@ -169,6 +169,14 @@
     :retract-fn (fn [count retracted] (dec count))
     :combine-fn +}))
 
+(defn exists
+  "Returns an accumulator that accumulates to true if at least one fact
+   exists and nil otherwise, the latter causing the accumulator condition to not match."
+  []
+  (assoc (count) :convert-return-fn (fn [v]
+                                      (when (pos? v)
```

Comment:
You may want a small impl-detail comment here that mentions why this needs to return true or nil instead of just being `:convert-return-fn pos?` (returning false sometimes).